play-json/shared/src/main/scala/play/api/libs/json/Writes.scala

```diff
@@ -26,17 +26,21 @@ trait Writes[-A] {
    * Returns a new instance that first converts a `B` value to a `A` one,
    * before converting this `A` value into a [[JsValue]].
    */
-  def contramap[B](f: B => A): Writes[B] = Writes[B](b => this.writes(f(b)))
+  def contramap[B](f: B => A): Writes[B] = Writes[B](b => self.writes(f(b)))
```

Comment:
Just out of interest, what's the reason for this set of changes?

Reply:
Use `self` as elsewhere in the class

Reply:
I guess you mean this makes it consistent with the use of `self` in the `Reads` class.  👌